Document behavior in case of suspension by owner. (#9477)
github/docs: content/developers/apps/managing-github-apps/suspending-a-github-app-installation.md

People who have installed a GitHub App, also called installation owners, can only suspend or unsuspend a GitHub App through their app's installation settings. Installation owners cannot use the API to suspend or unsuspend their app installation.

> {% note %}
> 
> **Note:** If an installation has been suspended by the GitHub App owner, then the option to unsuspend the installation is removed from the installation owner's app's settings and thus they will not be able to unsuspend it on their own. They can still change other settings (e.g., repository selection) while the app is suspended.
> 
> {% endnote %}

Reviewer: This is general information that doesn't meet [our criteria for a callout](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#callouts). 

```suggestion
If an installation has been suspended by the {% data variables.product.prodname_github_app %} owner, installation owners cannot unsuspend their installations of the {% data variables.product.prodname_github_app %}. However, installation owners can change other settings, such as repository selection, while the app is suspended.
```

People who have installed a GitHub App, also called installation owners, can only suspend or unsuspend a GitHub App through their app's installation settings. Installation owners cannot use the API to suspend or unsuspend their app installation.

> Note that if an installation has been suspended by the GitHub App owner, then the option to unsuspend the installation is removed from the installation owner's app's settings and thus they will not be able to unsuspend it on their own. They can still change other settings (e.g., repository selection) while the app is suspended.
> 

Reviewer 2: ```suggestion
{% note %}

**Note:** If an installation has been suspended by the GitHub App owner, then the option to unsuspend the installation is removed from the installation owner's app's settings and thus they will not be able to unsuspend it on their own. They can still change other settings (e.g., repository selection) while the app is suspended.

{% endnote %}
```

People who have installed a GitHub App, also called installation owners, can only suspend or unsuspend a GitHub App through their app's installation settings. Installation owners cannot use the API to suspend or unsuspend their app installation.

{% note %}

**Note:** If an installation has been suspended by the GitHub App owner, then the option to unsuspend the installation is removed from the installation owner's app's settings and thus they will not be able to unsuspend it on their own. They can still change other settings (e.g., repository selection) while the app is suspended.

> 
> {% endnote %}
> {% data reusables.user-settings.access_settings %}

Reviewer 2: ```suggestion

{% endnote %}

{% data reusables.user-settings.access_settings %}
```
